examples/step-61/step-61.cc

```diff
@@ -0,0 +1,885 @@
+   // when we evaluate pressures. We will do the same way here. For the coefficients of the projection, we do matrix multiplication, i.e.,
+   // the inverse of the Gram matrix times the matrix with $\mathbf{K} \mathbf{w} \cdot \mathbf{w}$ as components.
+   // Then we multiply all these coefficients and name it to be @p beta. The numerical velocity is the product of coefficient
+   // @p beta and basis functions of the Raviart-Thomas space.
+   for (; cell!=endc; ++cell,++cell_rt)
+       {
+        fe_values_rt.reinit (cell_rt);
+        fe_values.reinit (cell);
+        cell_matrix_rt = 0;
+        cell_matrix_E = 0;
+        cell_matrix_D = 0;
+        cell_matrix_C = 0;
+        cell_matrix_F = 0;
+        velocity_cell = 0;
+        coefficient.value_list (fe_values_rt.get_quadrature_points(),
+                                coefficient_values);
+
+        // The component of this cell matrix is the integral of $\mathbf{K} \mathbf{w} \cdot \mathbf{w}$.
+        for (unsigned int q=0; q<n_q_points_rt; ++q)
+            {
+               for (unsigned int i=0; i<dofs_per_cell_rt; ++i)
+                   {
+            		 const Tensor<1,dim> phi_i_u = fe_values_rt[velocities].value (i, q);
+
+                     for (unsigned int j=0; j<dofs_per_cell_rt; ++j)
+                      {
+                       const Tensor<1,dim> phi_j_u = fe_values_rt[velocities].value (j, q);
+
+                       cell_matrix_E(i,j) += (coefficient_values[q] *
+                    		                   phi_j_u * phi_i_u *
+                    		                   fe_values_rt.JxW(q));
+                      }
+                   }
+                }
+
+        // This is the Gram matrix on the cell.
+         for (unsigned int q=0; q<n_q_points_rt; ++q)
+              {
+                for (unsigned int i=0; i<dofs_per_cell_rt; ++i)
+                    {
+                      const Tensor<1,dim> phi_i_u = fe_values_rt[velocities].value (i, q);
+                      for (unsigned int j=0; j<dofs_per_cell_rt; ++j)
+                          {
+                            const Tensor<1,dim> phi_j_u = fe_values_rt[velocities].value (j, q);
+                            cell_matrix_rt(i,j) += (
+                           		                   phi_i_u * phi_j_u *
+                           		                   fe_values_rt.JxW(q));
+                           }
+                     }
+                }
+
+         // We take the inverse of the Gram matrix, take matrix multiplication and get the matrix with coefficients of projection.
+          cell_matrix_rt.gauss_jordan();
+          cell_matrix_rt.mmult(cell_matrix_D,cell_matrix_E);
+
+         // This is to extract pressure values of the element.
+          cell->get_dof_indices (local_dof_indices);
+          cell_solution = 0;
+          for (unsigned int i=0; i<dofs_per_cell; ++i)
+              {
+             	cell_solution(i) = solution(local_dof_indices[i]);
+              }
```

Comment:
You are using this variable for the first time in line 756. Please move this piece of code to line 747 where you compute `beta`.